tools/GraphClust/CollectResultsNoAlign/glob_report_no_align.xml

```diff
@@ -0,0 +1,184 @@
+<tool id="glob_report_no_align" name="cluster_collection_report_no_align" version="0.5" >
+  <requirements>
+    <requirement type="package" version="0.6.0">graphclust-wrappers</requirement>
+    <requirement type="package" version='0.5'>perl-array-utils</requirement>
+    <requirement type="package" version='0.18.1'>scikit-learn</requirement>
+    <requirement type="package" version='1.8.10'>locarna</requirement>
+    <requirement type="package" version='2.1'>rnaz</requirement>
+    <requirement type="package" version="1.1.2">infernal</requirement>
+    <requirement type="package" version='2.2.10'>viennarna</requirement>
+    <requirement type="package" version='1.3.30'>graphicsmagick</requirement>
+    <requirement type="package" version='0.6.1'>rscape</requirement>
+    <requirement type="package" version='6.0'>unzip</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" />
+  </stdio>
+  <command>
+    <![CDATA[
+        unzip $FASTA  &> /dev/null &&
+
+          mkdir ./CMSEARCH &&
+          mkdir ./MODEL &&
+
+        #set $inputFiles = ""
+
+        #for $cms_res in $cmsearch_results:
+            ###set $inputFiles += str($cms_res.element_identifier)+','
+          ln -f -s  '$cms_res' ./CMSEARCH/$cms_res.element_identifier &&
+        #end for
+        #set $inputFiles = $inputFiles[:-1]
+
+        #set $inputFilesTrees = ""
+
+        #for $mods in $model_tree_files:
+            ###set $inputFilesTrees += str($mods.element_identifier)+','
+            ln -f -s  '$mods' ./MODEL/$mods.element_identifier &&
+        #end for
+        #set $inputFilesTrees = $inputFilesTrees[:-1]
+
+     
+        'glob_res.pl'
+                ##'$inputFiles'
+                $merge_cluster_ol
+                $merge_overlap
+                $min_cluster_size
+                $cm_min_bitscore
+                $cm_max_eval
+                1 ## cm_bitscore_sig
+                $partition_type ''
+                $cut_type
+                ##'$inputFilesTrees'
+                0 ## zero means do not align
+        #if  $iteration_num.iteration_num_selector:
+          $iteration_num.CI
+          $final_partition_soft
+          $final_partition_used_cmsearch
+          '$combined_cm'
+
+        #end if
+
+        &&
+        python '$__tool_directory__/evaluation.py' FASTA/ RESULTS/
+       
+    #if $cdhit:
+        &&
+        python '$__tool_directory__/addCdhitseqs.py' '$cdhit'
+      #end if
+]]>
+  </command>
+  <inputs>
+    <param type="data" name="FASTA" format="zip" help="FASTA.zip from pre-processing step"/>
```

Comment:
There was something special were going on that we had to do it like this, as I remember.

Reply:
Hmm, opened the file and now I remember. There are many meta files beside the fasta file for mapping ids, etc. We zipped them otherwise we had to have 5 extra pins from pre-processing and 5 extra input pins to this one.

Reply:
ok.